components/emoji_picker/components/emoji_picker_skin.tsx

```diff
@@ -67,29 +71,32 @@ export class EmojiPickerSkin extends React.PureComponent<Props, State> {
             const spriteClassName = classNames('emojisprite', `emoji-category-${emoji.category}`, `emoji-${emoji.unified.toLowerCase()}`);
 
             return (
-                <div
-                    className='skin-tones__icon'
+                <button
+                    className='style--none skin-tones__icon'
+                    type='button'
```

Comment:
any reason we need to specifically mention type button here

Reply:
Good eye! I can't see why it would need to be explicitly mentioned here, I'll try removing it and push up the change.